src/components/ErrorBoundary/ErrorBoundary.spec.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import ErrorBoundary from './ErrorBoundary';
+
+describe('<ErrorBoundary />', () => {
+  it('should generate an error message when an error is caught', () => {
```

Comment:
I don't think this test is very useful. Try to trigger an error by adding a buggy component as error boundary children